src/screens/CreatePost/index.js

```diff
@@ -791,7 +789,6 @@ const CreatePost = () => {
                 setIsPollMultipleChoice(ismultiplechoice)
               }
               expiredobject={postExpired[expiredSelect].expiredobject}
-              isAnonimity={typeUser}
             />
           )}
           <Gap style={styles.height(26)} />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The prop `isAnonimity` has been removed. If this prop was used in the child component, ensure it's no longer needed or is being provided through a different mechanism.
```diff
- isAnonimity={typeUser}
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->